src/database/queries/bulkUploads/selectWithPagination.sql

```diff
@@ -3,6 +3,7 @@ SELECT
   file_name as "fileName",
   source_key as "sourceKey",
   status as "status",
+	file_size as "fileSize",
```

Comment:
It looks like this is tab-indented while the surrounding SQL is space-indented.
```suggestion
  file_size as "fileSize",
```

Reply:
Ohhhhhhhhhhhhhh prettier did not update our .sql files apparently...

We face a choice: should we use spaces in sql and tabs everywhere else (in which case I'd add a line to the editorconfig) or should I update all the sql files to use tabs.

For now I am leaning towards having spaces in the sql files and we can always normalize it later if we want.